syntax = "proto2";

package proto.server;

message MsgPing {
    optional uint32 index = 1;
}

message MsgPong {
    optional string index = 1;
}

// 机器人测试消息
message MsgRobot {
    optional string test = 1;
}

// tcp测试包
message MsgTestTcpRequest {
    optional uint32 count = 1;
}

message MsgTestTcpReply {
    optional string test = 1;
}

message pair64 {
	optional uint32 key = 1;
	optional uint32 value = 2;
}

message map64 {
	repeated pair64 pair_list = 1;
}

